### ISSUE TITLE
Usability: The "Add Schema Property" dialog should offer the property attributes as well

### ISSUE THREAD

**Issue by 0815fox** [Apicurio/apicurio-studio#414]
From a usability POV adding new parameters should be as fluent as possible IMO. It consumes too many clicks to create a new property, then click edit and then set the attributes "required", "type" and "formatted as" in another view. It would be much easier to simply have the attributes beside Name in the "Add Schema Property" dialog.

**Comment by EricWittmann** [comment]
Agreed - we're actually  moving the UX in this direction already and, although we haven't specifically re-designed the user flow for Add Schema Property yet, your approach is consistent with what we've been doing for other entities.  So yeah - there should be some changes in this area soon and I think the flow will be much improved!
